### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by rexemtoxa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,58 +236,173 @@ This a simple example on how config should look like. Please go through README t
...
+    { category: 'Date of issue', value: '2021-02-15' },
+    { category: 'Document number', value: '4114414141' },
+    { category: 'Email', value: 'email@email.com' },
+    { category: 'Gender', value: 'male' },
+    { category: 'Nationality', value: 'AZE' },
+    { category: 'Personal number', value: '123123123123' },
+  ],
+};
 ```
+In case when you not specify the Form field but specify this data into profile, this data will send from web sdk to the server and will participate in cross-checking
```
Comment (lukesolo) [inline README.md:317]: Form fields and pre-filled fields will be sent to the server and will be compared with extracted data from a document.

File: README.md [hunk 2]
```diff
@@ -141,58 +236,173 @@ This a simple example on how config should look like. Please go through README t
...
+
+We provide cross-checking comparison by next Form fields:
```
Comment (lukesolo) [inline README.md:319]: Profile checking is provided for those fields:

File: README.md [hunk 3]
```diff
@@ -141,58 +236,173 @@ This a simple example on how config should look like. Please go through README t
...
+* Nationality
+* Nationality code
+* Document number
+* Place of birth
+* Place of issue
 
 ### Visual Appearance
 
-SHADOW DOM
+It is possible to customize the styles by two way:
```
Comment (lukesolo) [inline README.md:336]: ways

File: README.md [hunk 4]
```diff
@@ -141,58 +236,173 @@ This a simple example on how config should look like. Please go through README t
...
+- By insert the custom css as string
```
Comment (lukesolo) [inline README.md:337]: inserting the custom CSS string

File: README.md [hunk 5]
```diff
@@ -141,58 +236,173 @@ This a simple example on how config should look like. Please go through README t
...
+- By CSS Variables
```
Comment (lukesolo) [inline README.md:338]: using CSS variables

File: README.md [hunk 6]
```diff
@@ -141,58 +236,173 @@ This a simple example on how config should look like. Please go through README t
...
 
-1. CSS Variables
-2. cssString
 
+To insert the custom css just specify filed *injectCSS* into config with a css string as a value
```
Comment (lukesolo) [inline README.md:341]: Example of custom CSS string:

File: README.md [hunk 7]
```diff
@@ -141,58 +236,173 @@ This a simple example on how config should look like. Please go through README t
...
+example
+```js
+const config = {
+  apiUrl: 'YOUR_URL',
+  containerId: 'getid-component',
+  flowName: 'yourFlowName',
```
Comment (lukesolo) [inline README.md:347]: CAPS_CASE

File: README.md [hunk 8]
```diff
@@ -141,58 +236,173 @@ This a simple example on how config should look like. Please go through README t
...
+  injectCSS: 'p {color: red; text-align: center;}'
+}
+```
+The using custom variables it's the more flexible and convenient way
```
Comment (lukesolo) [inline README.md:351]: Custom variables is a more flexible and convenient way. List of variables:

File: README.md [hunk 9]
```diff
@@ -141,58 +236,173 @@ This a simple example on how config should look like. Please go through README t
...
+--getid-input-width
+--getid-transition
+--getid-font-family
+--getid-background-color
+--getid-max-width
+--getid-min-width
+--getid-border-radius
+--getid-input-border-radius
+```
+to customize style specify this variables for element with next id: getid-main, getid-popup__main
```
Comment (lukesolo) [inline README.md:383]: Use these element ids (`getid-main`, `getid-popup__main`) to customize style using variables

**Review by lukesolo — CHANGES_REQUESTED** [review]

File: README.md [hunk 10]
```diff
@@ -64,11 +62,11 @@ In order to make it work you will need access to GetID api. Please get in touch
...
 
-ADD LINKS
 
 Both can be found and modified either through your GetID admin panel or via contacting our 
 [integration support](mailto:support@getid.ee?subject=[GitHub]%20Obtaining%20GetID%20credentials).
 
+[Contacts](https://getid.ee/contact-us/)
 ## Installation
 
 #### Including the library
```
[the remark this answers is not in the document]
Comment (rexemtoxa) [reply README.md:72]: done

File: README.md [hunk 11]
```diff
@@ -82,38 +80,135 @@ The latest stable build can be found in example snippet below.
...
 
-ADD INFO ABOUT NPM
 
 Script loads the widget via window object along with sdk config and jwt token.
 ```js
 window.getidWebSdk.init(config);
 ```
-ADD EXAMPLES WITH JWT AND SDK KEY
+
+- NPM launcher
```
[the remark this answers is not in the document]
Comment (rexemtoxa) [reply README.md:89]: done

File: README.md [hunk 12]
```diff
@@ -82,38 +80,135 @@ The latest stable build can be found in example snippet below.
...
+        console.log(error);
+    },
+};
+
+init(config);
+
+```
+You can specify the sdkKey options, or you can get the JWT token and pass it for authorization
+
+config example for authorization by sdkKey:
```
[the remark this answers is not in the document]
Comment (rexemtoxa) [reply README.md:131]: done

File: README.md [hunk 13]
```diff
@@ -82,38 +80,135 @@ The latest stable build can be found in example snippet below.
...
+  }],
+  onComplete({ id }) {
+      alert(id);
+  },
+  onFail(error) {
+      console.log(error);
+  },
+};
+```
+configuration example with authorization by JWT
```
[the remark this answers is not in the document]
Comment (rexemtoxa) [reply README.md:155]: done

File: README.md [hunk 14]
```diff
@@ -82,38 +80,135 @@ The latest stable build can be found in example snippet below.
...
+
+```js
+const jwt = (await fetch(`${apiUrl}/sdk/v2/token`, {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json',
+    'Access-Control-Allow-Origin': '*',
```
[the remark this answers is not in the document]
Comment (rexemtoxa) [reply README.md:162]: done

**Pull request merged**